pkg/selinux/views/setroubleshoot-view.js

```diff
@@ -1,4 +1,4 @@
-/* DO NOT EDIT. Automatically generated file
+/*
```

Comment:
Is this file not automatically generated anymore? And how did you generate it? There are a lot of trailing spaces that I didn't seem to get when using jsx.